build/crds/devices/devices_v1beta1_device.yaml

```diff
@@ -755,6 +755,12 @@ spec:
             description: DeviceStatus reports the device state and the desired/reported
               values of twin attributes.
             properties:
+              last_online:
```

Comment:
Please use camel case, and add a suffix 'Time' makes the field more explicit, like 'lastOnlineTime'.

Reply:
This name has been used in many places in the past. If want to change it, may have to change a lot of things. Need to start with the edge sqlite table structure.

<img width="613" alt="image" src="https://github.com/user-attachments/assets/d4e146a0-a670-4db2-9e51-b29ae090a620">

Reply:
+1 :)